tests/db/test_orm.py

```diff
@@ -2,7 +2,9 @@
 import unittest
 
 from remoteappmanager.db import orm
-from remoteappmanager.db.orm import Database, transaction, Accounting
+from remoteappmanager.db.orm import (Database, transaction, Accounting,
+                                     AccountingInterface)
```

Comment:
We might consider naming the ABCAccounting to `User` or `AppAccounting`. We should revisit naming on Friday